Automattic/wp-calypso
# Signup: Remove clickable domain row AB test

This PR makes the clickable row the default implementation, removing the A/B test.

![screen shot 2016-11-11 at 17 21 05](https://cloud.githubusercontent.com/assets/418473/20227404/4cd57dd8-a833-11e6-971a-5369a9f6f6f6.png)

To test:

- Go through a signup flow that has domains (e.g. `main`)
- Type something in the domain search bar
- Verify that list has clickable rows, and that they're rendering/working normally
- Open A/B test at "Dev" menu
- Verify `domainSuggestionClickableRow` is gone
